example/pubspec.lock

```diff
@@ -150,7 +150,7 @@ packages:
       name: material_color_utilities
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.1.2"
+    version: "0.1.3"
```

Comment:
Is this change needed as you didn't touch .yml please revert. This file should be updated in sync with upgrading dependencies

Reply:
done